src/app/formatters/HelpTextFormatter.java

```diff
@@ -6,7 +6,7 @@ public String format() {
       return "Vaccination register options:\n"
             + String.format("%-110s %s\n", "list (patients | doctors | vaccines | hospitals | hospitals_doctors | registered_vaccinations) [id]", "Lists records in selected entity")
             + String.format("%-110s %s\n", "delete_patient (patient_id)", "Deletes patient and it's registrations from the registry")
-            + String.format("%-110s %s\n", "cancel_registration (patient_id)", "Deletes registered vaccination from registry")
```

Comment:
This was correct. This serves to delete single registered vaccination, not all registered vaccinations of a patient

Reply:
So perhaps the command name is wrong, but the command description should be ok.